src/test/java/net/revelc/code/formatter/xml/XMLFormatterTest.java

```diff
@@ -24,18 +24,21 @@
 
 /**
  * @author yoshiman
+ * @author jam01
  */
 public class XMLFormatterTest extends AbstractFormatterTest {
 
     @Test
     public void testDoFormatFile() throws Exception {
-        // FIXME Handle linux vs windows since this formatter does not accept line endings
+        // Since we set the line endings via options for xml, we cannot rely on CRLF inside doTestFormat.
```

Comment:
@jam01 Trivial thing here.  Can you note xml formatter rather than json formatter (copy/paste gotcha)?

Reply:
:heavy_check_mark: